### PR TITLE
updated copyright

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by ralphtheninja** [comment]
@rvagg About the license. Should we keep the MIT +no-false-attribs license or change to plain MIT like was done for e.g. `node-leveldown` et. al.?

**Comment by rvagg** [comment]
Switch to MIT but requires agreement from other collaborators, and there are a lot of them now. Possibly propose it as a separate issue and aim for no -1s.

**Comment by ralphtheninja** [comment]
@rvagg Sure. https://github.com/rvagg/learnyounode/issues/186

**Comment by rvagg** [comment]
actually @ralphtheninja, can you update this change to assign copyright to all contributors like we do in level* packages? I just noticed it's still assigned to me only.

**Comment by ralphtheninja** [comment]
Np. Just gonna grab a coffee and walk the dog first :)

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2014, Rod Vagg (the "Original Author")
+Copyright 2013-2015, Rod Vagg (the "Original Author")
```
Comment (rvagg) [inline LICENSE]: assignment needs to be fixed here too, have a look at the LICENSE.md (note markdown!) in levelup, we can just copy that across.

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 All rights reserved.
 
 MIT +no-false-attribs License
```
Comment (rvagg) [inline LICENSE]: this too

**Comment by rvagg** [comment]
ugh, a mess, but here's the thing about the current copyright: it's assigned to me so I should just unilaterally change it to MIT and reassign to the collaborators! I might close this and fix up a new one @ralphtheninja

**Pull request closed without merging**